CONTRIBUTING.md

```diff
@@ -1,4 +1,49 @@
+The brief summary below outlines contribution guidelines specific to the OpenSearch-Security Repo.
+
+## Triaging
+
+OpenSearch-Security seeks to promote an inclusive and engaged community of contributors. In order to facilitate this, weekly triage meetings are open-to-all and attendance is encouraged for anyone who hopes to contribute, discuss an issue, or learn more about the project.
+
+FAQ about OpenSearch-Security Triage Meetings:
+
+### Do I need to attend for my issue to be addressed/triaged?
+
+Attendance is not required for your issue to be triaged or addressed. All new issues are triaged weekly.
+
+### How do I join the Backlog & Triage meeting?
+  
+Meetings are hosted regularly at 3 PM Eastern Time (Noon Pacific Time) and can be joined via the links posted on the [Upcoming Events](https://opensearch.org/events) webpage.
+
+After joining the Zoom meeting, you can enable your video / voice to join the discussion.  If you do not have a webcam or microphone available, you can still join in via the text chat.
+    
+If you have an issue you'd like to bring forth please consider getting a link to the issue so it can be presented to everyone in the meeting.
+
+### Is there an agenda for each week? 
+
+Meetings are lightly structured such that new issues are reviewed first, then the sprint backlog, and finally the backlog. There is no specific ordering within each category. 
+
+If you have an issue you would like to discuss but do not have the ability to attend the entire meeting please attend when is best for you and signal that you have an issue to discuss when you arrive. 
+
+### Do I need to have already contributed to the project to attend a triage meeting?
+ 
+No all are welcome and encouraged to attend. Attending the Backlog & Triage meetings is a great way for a new contributor to learn about the project as well as explore different avenues of contribution. 
+
+### Is this meeting a good place to get help setting up security features on my OpenSearch instance?
+
+While we are always happy to help the community, the best resource for implementation questions is [the OpenSearch forum](https://forum.opensearch.org/c/security/3).
+
+There you can find answers to many common questions as well as speak with implementation experts. 
+
+### Is this where I should bring up potential security vulnerabilities?
+
+Due to the sensitive nature of security vulnerabilities, please report all potential vulnerabilities directly by by following the steps outlined on the [SECURITY.md](https://github.com/opensearch-project/security/blob/main/SECURITY.md) document.
+
+### Who should I contact if I have further questions? 
+  
+You can reach out to any member of the OpenSearch-Security team but specific contacts include Stephen Crawford (steecraw@amazon.com), Dave Lago (davelago@amazon.com), and Peter Nied (petern@amazon.com).
```

Comment:
how would people reach out to any member of the OpenSearch-Security team? If we are not going to provide contact info from all mbmers, I would remove that first sentence and directly say to feel free to email any of us 3.

Reply:
Rather than direct users to contact maintainers offline, what about having them create an issue in the repository?

Reply:
Something that could be followed up on another pull request is adding a 'triage discussion' issue template that helps structure and direct questions to be addressed during the triage meeting.  See our existing bug report templates for how we do this today

https://github.com/opensearch-project/security/blob/a57fd0a745a398b1fb7ba005bda44d0e55f5fc88/.github/ISSUE_TEMPLATE/bug_report.md?plain=1#L1-L31

Reply:
Yeah I think that makes sense. I just feel like it is nice to be able to contact someone directly as opposed to having to create an issue for every little issue especially if someone has a question about issues themselves.